packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { Localized } from 'fluent-react';
+import { formatCurrencyInCents } from '../../lib/formats';
+
+import './index.scss';
+import { Plan, Profile } from '../../store/types';
+
+type PlanDetailsProps = {
+  profile: Profile;
+  plan: Plan;
+  showExpandButton: boolean;
+  className?: string;
+};
+
+export const PlanDetails = ({
+  profile,
+  plan,
+  showExpandButton,
+  className = 'default',
+}: PlanDetailsProps) => {
+  const initialDetailsHiddenState = showExpandButton ? true : false;
+  const [detailsHidden, setDetailsState] = useState(initialDetailsHiddenState);
+  const { displayName, email } = profile;
+  const {
+    // plan_id: string;
+    plan_name,
+    plan_metadata,
+    // product_id: string;
+    product_name,
+    product_metadata,
+    currency,
+    amount,
+    interval,
+  } = plan;
+
+  // const iconURL = product_metadata ? product_metadata.webIconURL : 'none.jpg';
+
+  console.log('PROFILE', profile);
+  console.log('PLAN', plan);
+
+  const up = <span className="up-chevron">&rsaquo;</span>;
+  const down = <span className="down-chevron">&rsaquo;</span>;
+
+  return (
+    <section className="container card plan-details-component">
+      <header>
+        <div className="plan-details-header-wrap">
+          <img
+            src={'/static/media/firefox-logo.16821f55.svg'}
+            alt={product_name}
+          />
+          <div className="plan-details-heading-wrap">
+            <p className="plan-details-product">{product_name}</p>
+            <p className="plan-details-description">Full-device VPN</p>
+          </div>
+        </div>
+        <p>
+          ${formatCurrencyInCents(amount)}/{interval}
+        </p>
+      </header>
+      {!detailsHidden ? (
+        <section className="plan-details-list" aria-expanded={!detailsHidden}>
+          <Localized id="plan-details-header">
+            <h4>Product details</h4>
+          </Localized>
+          <ul>
+            <li>something abt the product</li>
+            <li>something abt the product</li>
+            <li>something abt the product</li>
+            <li>something abt the product</li>
+            {/* unordered list generated
+            over a loop with each description details
+          */}
+          </ul>
+        </section>
+      ) : null}
+      {showExpandButton ? (
+        <section className="footer">
```

Comment:
If we're going to use `<header>`, we should use `<footer>`.

Normally though, there's content inbetween `<header>` and `<footer>` and there isn't any on initial load... Looking at the wires, I agree (`.plan-details-component`) should be its own `section`. We could remove the `<header>` grouping ([this article](http://html5doctor.com/avoiding-common-html5-mistakes/) goes into details on section/header overuse) _but_ I could see how "show/hide details" could be part of the `<footer>` for that section. I feel more strongly about `.plan-details-list` just being a regular `div` though.

Looking at the wires I think `Set up your subscription` can remain the `h2` for the page, but man it's a little awkward moving this entire section out and to the right in desktop (and hiding the show/hide buttons). The product name (`.plan-details-product`) should probably be an h3 and "Product details" an h4 (which is how you have it), and since this is self-contained in its own `section`, "Billing information" can remain an h3.

Reply:
I went with `.footer` here because of the global css on the page for the `<footer>` element was making it hard to reset. I'm happy to remove header/footer references